app/services/translation/masker.py

```
@@ -0,0 +1,47 @@
+from typing import Dict, Tuple
+
+class ProtectedWordMasker:
+    def __init__(self, registry_path: str):
+        self.registry_path = registry_path
+        self._load_registry()
+
+    def _load_registry(self):
+        try:
+            with open(self.registry_path, "r", encoding="utf-8") as f:
+                data = json.load(f)
+        except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)